### ISSUE TITLE
[Idea] LinkedIn

### ISSUE THREAD

**Issue by Sjoerd** [toolkito/laravel-social-auto-posting#6]
Will LinkedIn support be added in the future?

**Comment by alihesari** [comment]
@Sjoerrdd, Yes, there is a possibility to add this feature. But my time is limited and I will try to do it as soon as possible. If you want to help expand this package, you can do the following.

Fork, then clone the repo:
```bash
git clone git@github.com:your-username/laravel-social-auto-posting.git
```
Make your changes and push to your fork and submit a pull request.

Thanks for your interest.
